Attendance_app.py

```diff
@@ -1,14 +1,14 @@
 import os
 import sys
+import pyautogui
 import subprocess
 from datetime import datetime
 import tkinter as tk
-from tkinter import messagebox, END

```

Comment:
Use **`messagebox`** from **`tkinter`** in the import itself and use **`messagebox`** variable directly in the code (wherever used) as currently done in the master.
Some IDEs fail to recognize the function in the tkinter library, thus indicating a warning.